templates/cronjob.yaml

```diff
@@ -0,0 +1,202 @@
+{{- if .Values.garbageCollect.enabled }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "docker-registry.fullname" . }}-garbage-collector
+  labels:
+    app: {{ template "docker-registry.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  schedule: {{ .Values.garbageCollect.schedule | quote }}
+  jobTemplate:
+    spec:
```

Comment:
Missing `spec.jobTemplate.metadata` block -- jobs created will be missing labels.

In all fairness, the labels on resources across this chart aren't consistent with one another anyway and could use a round of cleanup.

Soft suggestion to add metadata here with consistent labels, but because of the above, not a blocking issue for me.

Reply:
copying https://github.com/ddelange/docker-registry.helm/blob/d78c010ca556a6bfc7650b575180ddf4d0fadc44/templates/deployment.yaml#L22-L34